localisation/00_zuwc_united_localisation.csv

```diff
@@ -2920,12 +2920,13 @@ EVTOPTA_WCPRM_600;As water wish.;;Wie Wasser w
 EVTOPTB_WCPRM_600;No! You have completely lost your hearing, old man!;;Nein! Ihr habt vollkommen das Geh�r verloren, Alter!;;;;;;;;;;;x
 EVTDESC_WCPRM_605;The oracle frowns and cries: "I'm the voice of the water, young one. The rivers want to warn you. If you decide to remain deaf and go against the tide, well, it is not in my power to convince you.";;Das Orakel runzelt die Stirn und schreit: �Ich bin die Stimme des Wassers, J�ngling. Die Fl�sse wollen dich warnen. Wenn du dich entscheidest, taub zu bleiben und gegen den Strom zu schwimmen, nun, dann liegt es nicht in meiner Macht, dich zu �berzeugen.�;;;;;;;;;;;x
 EVTOPTA_WCPRM_605;Old fool!;;Alter Narr!;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+pick_titanic_branch;Choose Titanic Way;;;;;;;;;;;;;x
+pick_titanic_branch_desc;Choose the way that you will follow in the name of the creators.;;;;;;;;;;;;;x
+picked_branch_cooldown;Recently Picked Religious Branch;;;;;;;;;;;;;x
+EVTDESC_WCREL_6000;The makers have created us different so that we are strong together: ancient titan-forged have sculpted mountains, dripped seas, built giant complexes around the world! It's time to find my purpose on Azeroth. May the titans guide me!;;;;;;;;;;;;;x
+EVTOPTA_WCREL_6000;Storms, heed my call!;;;;;;;;;;;;;x
+EVTOPTB_WCREL_6000;Let's roll these rocks!;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTOPTB_WCREL_6000;Time to shape stone!;;;;;;;;;;;;;x
```

Reply:
SAD, wanted to make reference to rock & roll or at least rolling stones.

Reply:
> SAD, wanted to make a reference to rock & roll or at least rolling stones.
```suggestion
EVTOPTB_WCREL_6000;A portion of this rock.. A pinch of these rolling stones.. Now quake as the earth binds you!;;;;;;;;;;;;;x
```

Reply:
@Khama-Llama can we shorten it? That's how it looks in-game.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/82837359-d6611580-9ed9-11ea-9634-32da93aeadbb.png)

</details>

Reply:
check again pls. shortened it a little, it fit's the box now